dev/ssl_darwin_test.go

```diff
@@ -0,0 +1,76 @@
+package dev
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	. "github.com/puma/puma-dev/dev/devtest"
+	"github.com/puma/puma-dev/homedir"
+	"github.com/stretchr/testify/assert"
+)
+
+func deleteAllPumaDevCAFromDefaultKeychain() {
+	forEachPumaDevKeychainCertSha := func(subCommand string) string {
+		return fmt.Sprintf(`for sha in $(security find-certificate -a -c "Puma-dev CA" -Z | awk '/SHA-1/ {print $3}'); do %s; done`, subCommand)
+	}
+
+	if err := exec.Command("sh", "-c", forEachPumaDevKeychainCertSha("security delete-certificate -Z $sha")).Run(); err != nil {
```

Comment:
given there might be a few certs with the same common name (because you ran `setup` several times or something) we need to find all the hashes and delete them that way.